resdk/resources/sample.py

```diff
@@ -32,7 +35,21 @@ def __init__(self, slug=None, id=None,  # pylint: disable=redefined-builtin
 
         self.presample = presample
 
-        BaseCollection.__init__(self, slug, id, model_data, resolwe)
+        super(Sample, self).__init__(slug, id, model_data, resolwe)
+
+    def update(self):
+        """Clear cache before updating."""
```

Comment:
The first line should describe the functionality. Specifics should follow below.